sydent/types.py

```diff
@@ -0,0 +1,3 @@
+from typing import Any, Dict
```

Comment:
Please put a license header at the top of this file (pretty much copy one from another file and ensure the year copyright year is correct).